redaxo/src/core/lib/util/file.php

```diff
@@ -194,9 +194,17 @@ public static function move(string $srcfile, string $dstfile): bool
     public static function delete($file)
     {
         return rex_timer::measure(__METHOD__, static function () use ($file) {
-            if (is_file($file)) {
-                return unlink($file);
+
+            $tryUnlink = @unlink($file);
```

Comment:
Ich würde das äußere `is_file` belassen. Wenn die Datei bereits nicht existiert, löschen wir dann nicht unnötig den stat-cache. (`unlink` liefert false für nicht existente Dateien).

Reply:
das das file bereits gelöscht ist, ist sehr unwahrscheinlich.
der zusätzliche is_file() call im common case hingegen wäre oft unnötig

Reply:
der stat cache beinhaltet übrigens immer nur max. 1 datei. dass dieser wirksam wird ist selten.

https://tideways.com/profiler/blog/the-php-stat-cache-explained

Reply:
Ah interessant! Das wir mir nicht bewusst.